corehq/apps/users/views/mobile/users.py

```diff
@@ -789,6 +792,26 @@ def create_mobile_worker(self, in_data):
         }
 
 
+@location_safe
```

Comment:
I think this should probably be behind a superuser flag (or maybe even a support toggle). because of the complexity and inconsistency of how restoring deleted users actually works it seems unwise to expose this to everyone (even if it is a magic url)